lib/aur-chroot

```diff
@@ -88,15 +86,12 @@ while read -r key _ value; do
     esac
 done < <(pacman-conf --config "$pacman_conf")
 
-if (( create )); then
-    # parent path is not created by mkarchroot (#371)
-    if [[ ! -d $directory ]]; then
-        sudo install -d "$directory" -m 755 -v
-    fi
+# parent path is not created by mkarchroot (#371)
+if [[ ! -d $directory ]]; then
```

Comment:
Shouldn't this be moved under `prepare` as before?

Reply:
Would make sense 👍